### PR TITLE
Hide delete button

### PR DESCRIPTION
**Pull request by swalsh15**
Only shows delete button if user is logged in and the users Id matches the Id of the user who posted the comment.

### CODE REVIEW

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 1]
```diff
@@ -100,11 +108,12 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
       response.sendRedirect("/index.html");
       return;  
     }
-
+    
     Entity commentEntity = new Entity("Comment"); 
     commentEntity.setProperty("message", comment);
     commentEntity.setProperty("name", name);   
     commentEntity.setProperty("picture", pictureURL);
+    commentEntity.setProperty("posterId", key);
```
Comment (g-harel) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:116]: Nit: I find it slightly confusing that key/id is used to refer to the same thing in different places.

File: portfolio/src/main/webapp/script.js [hunk 2]
```diff
@@ -124,4 +134,18 @@ function uploadComment() {
...
+function deleteComment(id) {
+  fetch('/delete-comment', {
+    method: 'POST',
+    body: "id=" + id,
+    headers: {
+      "Content-Type": "application/x-www-form-urlencoded"
+    }
+  }).then(function(response) {
+    if (!response.ok) {
+      // failure
```
Comment (g-harel) [inline portfolio/src/main/webapp/script.js:148]: `console.error()` would be appropriate here if you don't show anything in the ui.

**Comment by swalsh15** [comment]
Changed the hide to be based on email instead of google user id because that id changes for every session.

File: portfolio/src/main/webapp/script.js [hunk 3]
```diff
@@ -131,7 +131,7 @@ function uploadComment() {
     }
   }).then(function(response) {
     if (!response.ok) {
-      // TODO: handle case when verification fails
+      console.error();
```
Comment (g-harel) [inline portfolio/src/main/webapp/script.js:134]: This func behave similar to `log`, you can add a hardcoded error cause or read from the response.

**Pull request merged**